### ISSUE TITLE
Validate oneof with values containing space

### ISSUE THREAD

**Issue by muhammadkholidb** [go-playground/validator#525]
### Package version eg. v8, v9: 

v9

### Issue, Question or Enhancement:

Is it possible to use **oneof** validation with values containing space character? Tried with single quote but it did not seem to work.

### Code sample, to showcase or reproduce:

```go

type Data struct {
    Status string `json:"status" validate:"required,oneof='Awaiting Verification' 'Verified' 'Failed Verification'"`
}

```

**Comment by deankarn** [comment]
@muhammadkholidb no, unfortunately, this isn't supported for `oneoff` at the moment.

As a workaround for now though is you can register your own custom validation function which accomplishes the same thing :)

**Comment by JonathanWThom** [comment]
Hello 👋 , I'm going to take a crack at this.

**Comment by fr3fou** [comment]
Isn't this closed by #541
